devsite/requirements.txt

```diff
@@ -28,6 +28,7 @@ django-countries==4.0
 django-filter==0.11.0
 django-webpack-loader==0.5.0
 django-model-utils==2.3.1
+jsonfield==2.0.2
 
 ##
 ## Open edX package dependencies
```

Comment:
This might be out of scope for this PR, since it's conflating devsite and Figures compat, but how are you planning to handle changes here based on the Open edX version? You list at least one Open edX dependency -- how will a dev who's developing for Ficus, say, install a requirements for the devsite that lines up with the version she's developing against? Does that matter?

Reply:
@abeals Really good point. I did this in haste,  and not out of scope for the PR. Hawthorn uses v2.02. Ginkgo uses v1.0.3. Specific to jsonfield, what matters is if the behavior and interfaces are identical between the two versions.

In general though, Figures is going to have to address this for some Python package incompatibility. A prime candidate is django-filter. So I need to do some thinking of how we'll address incompatible packages between Open edX releases. One avenue is to checkout other commonly used Django packages that have third party package dependencies and see how they address it. In a "not so worst case scenario" Figures can

A) loosen up the versions specified in the requirements file(s)
B) have different requirements files to target different named releases of Open edX

This will stave off the need for different release branches.

In a worst case scenario, Figures can implement "six-style" dependency packages that encapsulate the conflict bearing functionality so we'd have a `figures-deps-ginkgo` and `figures-deps-hawthorn` "mini-package" that serves as an adapter for one or more core functional areas, like view filtering as is the case with `django-filter`. This way Figures can then use approach B above.

Sounds like a spike. Interested in your thoughts on my ideas ^^^.

cc @OmarIthawi , @melvinsoft , @bryanlandia, @ lpm0073